fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/type/DateParmBehaviorUtil.java

```diff
@@ -0,0 +1,286 @@
+/*
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.util.type;
+
+import static com.ibm.fhir.persistence.jdbc.JDBCConstants.AND;
+import static com.ibm.fhir.persistence.jdbc.JDBCConstants.BIND_VAR;
+import static com.ibm.fhir.persistence.jdbc.JDBCConstants.DATE_END;
+import static com.ibm.fhir.persistence.jdbc.JDBCConstants.DATE_START;
+import static com.ibm.fhir.persistence.jdbc.JDBCConstants.DATE_VALUE;
+import static com.ibm.fhir.persistence.jdbc.JDBCConstants.DOT;
+import static com.ibm.fhir.persistence.jdbc.JDBCConstants.LEFT_PAREN;
+import static com.ibm.fhir.persistence.jdbc.JDBCConstants.RIGHT_PAREN;
+import static com.ibm.fhir.search.date.DateTimeHandler.generateTimestamp;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.List;
+
+import com.ibm.fhir.persistence.jdbc.JDBCConstants.JDBCOperator;
+import com.ibm.fhir.search.SearchConstants.Prefix;
+import com.ibm.fhir.search.parameters.QueryParameter;
+import com.ibm.fhir.search.parameters.QueryParameterValue;
+
+/**
+ * <a href="https://hl7.org/fhir/search.html#date>FHIR Specification: Search
```

Comment:
missing the closing `"`
```suggestion
 * <a href="https://hl7.org/fhir/search.html#date">FHIR Specification: Search
```

Reply:
whoops... I'll add not my next commit